packages/world/src/systems/MineSystem.sol

```diff
@@ -113,15 +113,15 @@ contract MineSystem is System {
     } else if (mineObjectTypeId.isSeed()) {
       _requireSeedNotFullyGrown(entityId);
     } else if (mineObjectTypeId == ObjectTypes.AnyOre) {
-      mineObjectTypeId = RandomOreLib._mineRandomOre(entityId, coord);
+      mineObjectTypeId = RandomResourceLib._collapseRandomOre(entityId, coord);
     }
 
+    uint128 finalMass = MassReductionLib._processMassReduction(caller, entityId);
+
     // First coord will be the base coord, the rest is relative schema coords
     Vec3[] memory coords = mineObjectTypeId.getRelativeCoords(baseCoord, Orientation._get(entityId));
 
-    uint128 finalMass;
     {
```

Comment:
can these curly braces be removed now? it doesnt seem like the scope has no variable instantiations anymore